palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/service/SimplePalisadeService.java

```diff
@@ -50,6 +49,12 @@
  * registerDataRequest. </p>
  */
 public class SimplePalisadeService implements PalisadeService {
+
+    /**
+     * Service name
+     */
+    public static final String NAME = "simple-service";
```

Comment:
"palisade-service" ? maybe "simple-palisade-service"

Reply:
I think this should actually sit in the PalisadeService class and be inherited here such that it is actually named "palisade-service"

Reply:
Moved and given "palisade-service" value